modules/manage/pages/cluster-maintenance/nodewise-partition-recovery.adoc

```diff
@@ -0,0 +1,40 @@
+= Node-wise Partition Recovery
+:description: Feature to recover partitions that have lost a majority of replicas.
+
+Multi-node or entire-AZ failures (especially in cloud environments) and some forms of human error may result in ‘stuck’ partitions with fewer replicas than required to make a quorum. In these scenarios, some data loss may be unavoidable but admins need to recover what they can and move on.
+
+IMPORTANT: This is a last ditch measure when all other recovery options have failed. In some cases, no remaining replicas may exist for the partitions on the dead nodes. This recovery method is used in scenarios where you are already experiencing data loss - the goal is to stop the loss of any more data.
+
+Node-wise partition recovery allows you to unsafely recover at least a portion of your data using whatever replicas are remaining. All replicas are moved off the target nodes and allocated to healthy nodes. This process repairs partitions while draining the target nodes all in one step.
+
+== Perform the recovery operation
+
+Node-wise partition recovery is performed via the `rpk` command `rpk cluster partitions unsafe-recover`. This command includes an interactive prompt to confirm execution of the generated recovery plan as it is a destructive operation. When you trigger node-wise partition recovery, the partitions on the node are rebuilt in a best-effort basis. You may lose some data which was not yet replicated to the surviving partition replicas as a result of executing this operation.
+
+The syntax for this command is as follows:
+
+ rpk cluster partitions unsafe-recover --from-nodes 1,3,5
+
+The `--from-nodes` parameter accepts a comma-delineated list of dead node IDs you wish to recover the data from. The above example would perform recovery operations on nodes 1, 3, and 5. Redpanda will assess these nodes to identify which partitions lack majority. It will then create a plan to recover the impacted partitions and ask you to confirm it. You must respond `yes` to the prompt to continue with recovery.
```

Comment:
```suggestion
The `--from-nodes` parameter accepts a comma-delineated list of dead node IDs you wish to recover the data from. The above example would perform recovery operations on nodes 1, 3, and 5. Redpanda will assess these nodes to identify which partitions lack majority. It will then create a plan to recover the impacted partitions and prompt you to confirm it. You must respond `yes` to continue with recovery.
```